_posts/2018-02-28-smart-window-shades.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Smart Window Shades
+date: 2018-03-01 00:00:00
+categories: projects
+description: A web service that allows one to control their window shades using a website
+author: Ayush Goel
+author-image: https://avatars2.githubusercontent.com/u/10407155?s=460&v=4
+author-bio: First Year Computer Science Student at Rochester Institute of Technology
+author-email: axg4975@rit.edu
+author-social:
+  github: https://github.com/axg4975
+  linkedin: https://www.linkedin.com/in/ayush-goel-589ba215a/
+---
+# Summary
+Smart Shades is a web service that allows one to control window shades using a website. The hardware requirement is a Raspberry Pi (connected to the Internet), a Stepper Motor, housing for the stepper motor (3D printed), a Stepper Motor Driver (I used the Pololu A4988), a 10K Ohm Resistor, and a domain. The application uses python, Flask, OpenShift in the back end, Bootstrap, JavaScript, and Ajax. In short, there are two Flask apps running, one on [CSH domain](shades.csh.rit.edu) and one on the Pi. The flask app on CSH website has the JavaScript and Ajax to make requests to the Flask app on the Pi, which can then take care of changing the window shades.
```

Comment:
Be consistent with your capitalization. Only proper nouns should be capitalized.

Reply:
Also, _a_ CSH domain/_the_ CSH domain, respectively.